sdk/deepbook/src/deepbook_sdk.ts

```diff
@@ -277,31 +373,34 @@ export class DeepBook_sdk {
 	public placeLimitOrder(
 		token1: string,
 		token2: string,
+		client_order_id: string,
 		poolId: string,
 		price: number,
 		quantity: number,
+		self_matching_prevention: boolean,
 		isBid: boolean,
 		expireTimestamp: number,
 		restriction: number,
 		accountCap: string,
 	): TransactionBlock {
 		const txb = new TransactionBlock();
 		const args = [
-			txb.object(String(poolId)),
+			txb.object(poolId),
+			txb.pure(client_order_id),
 			txb.pure(Math.floor(price * 1000000000)), // to avoid float number
```

Comment:
The scalar for price is always going to be 10^9, maybe we should make a constant for this?